starter-code/css/main.css

```diff
@@ -6,3 +6,133 @@ Green: #00B172
 White: #FFF
 
 */
+* {
+  margin: 0;
+  padding: 0;
+  font-family: Helvetica,Arial,sans-serif;
+}
+
+.nav-bar {
+  background-color: white;
+  height: 75px;
+  width: 100%;
+  padding: 15px 10px 5px 10px;
+
+}
+ .nav-bar > ul {
+    width: 1200px;
+    margin: 0 auto;
+ }
+
+ .nav-bar li {
+   text-align: right;
```

Comment:
`text-align: right` AND `float: right` ? You could just delete `text-align: right` or the `float: right` .